SECURITY.md

```diff
@@ -0,0 +1,41 @@
+# Osmosis Bug Reporting and Feature Requests
+
+Osmosis uses GitHub to manage feature requests and bugs for its platform. This is done via GitHub Issues.
+
+## Feature Request
+
+For a feature request, please [create a GitHub issue](https://github.com/osmosis-labs/osmosis/issues/new). Clearly state your use case and what value it will bring to other users or developers on Osmosis.
+
+If it is something that can be handled by a parameter change, we encourage you to join the Osmosis Commonwealth Forum for discussions. You can participate in the forum by visiting https://gov.osmosis.zone/overview. Additionally, you can also join the Osmosis Discord community at https://discord.com/invite/osmosis and engage in the #governance-proposals channel to discuss and consider governance proposals.
+
+## Standard Priority Bug
+
+For a bug that is non-sensitive and/or operational in nature rather than a critical vulnerability, please add it as a GitHub issue.
+
+If it is not triaged in a couple of days, feel free to tag @p0mvn, @czarcas7ic, @mattverse, @daniel-farina, @ValarDragon, @AlpinYukseloglu 
+
+## Critical Bug or Security Issue
+
+If you're here because you're trying to figure out how to notify us of a security issue, please send us a message on https://security.osmosis.zone/ or via [Telegram](https://t.me/OsmoSec) 
+
+Please avoid opening public issues on GitHub that contain information about a potential security vulnerability as this makes it difficult to reduce the impact and harm of valid security issues.
+
+### Coordinated Vulnerability Disclosure Policy
+
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading Osmosis's services.
+
+## Vulnerability Disclosure Process
+
+1. **Report:** Report the vulnerability by sending a message to https://security.osmosis.zone/
```

Comment:
Does anyone check this? I'd rather keep messaging clear to Telegram if thats the only thing thats going to be checked

Reply:
+1